### ISSUE TITLE
FeedForward - Optimizer

### ISSUE THREAD

**Issue by Tiamaty** [arcedos/arcedosml#15]
Add an option so we can tune the optimizer parameter. Now, for example, we can not change the learning rate of the neural network.
